actions/watch.ts

```diff
@@ -36,34 +43,51 @@ const GPV2SETTLEMENT = "0x9008D19f58AAbD9eD0D60971565AA8510560ab41";
 export const checkForSettlement: ActionFn = async (
   context: Context,
   event: Event
+) => {
+  return _checkForSettlement(context, event).catch(handleExecutionError);
+};
+
+/**
+ * Asyncronous version of checkForSettlement. It will process all the settlements, and will throw an error at the end if there was at least one error
```

Comment:
```suggestion
 * Asynchronous version of checkForSettlement. It will process all the settlements, and will throw an error at the end if there was at least one error
```